wicker/core/storage.py

```diff
@@ -237,7 +237,7 @@ class WickerPathFactory:
 
     Our bucket should look like::
 
-        <dataset-root>
+        <optional_mount_path>/<dataset-root>
```

Comment:
IMHO this isn't really what the bucket looks like. I feel like what we should do here is add another short bit at the end of this section that explains what happens if you FUSE mount the bucket into your container or instance and use the `optional_mount_path`.